mitmproxy/addons/replace.py

```diff
@@ -38,7 +38,9 @@ def parse_hook(s):
         patt, a, b = parts
     else:
         raise exceptions.OptionsError(
-            "Invalid replacement specifier: %s" % s
+            "Invalid replacement specifier: {}\n"
+            "If you want to use mitmproxy in reverse proxy mode. "
+            "Use --mode reverse:SPEC instead".format(s)
```

Comment:
Right now, I'm calling the check function only if parse_args() throws an exception.  "-R" won't throw an exception in parse_args. So should we just call check function every time before calling parse_args?